static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -56,26 +59,59 @@ class ReleasesConfirm extends Component {
   render() {
     const { isLoading, percentage } = this.state;
     const {
-      pendingReleases,
       pendingCloses,
-      isProgressiveReleaseEnabled
+      isProgressiveReleaseEnabled,
+      separatePendingReleases
     } = this.props;
-    const releasesCount = Object.keys(pendingReleases).length;
     const closesCount = pendingCloses.length;
 
     const isPercentageValid = +percentage > 0 && +percentage <= 100;
 
+    const {
```

Comment:
Nit picking, but if these are passed from redux we could map them to individual promps in `mapStateToProps` so component doesn't rely on the format of the data.